config/schema.yaml

```diff
@@ -495,19 +533,22 @@ schema:
       -
         name: instrument
         type: string
+        length: 8
         primary_key: true
         doc: >
           Name of the Instrument with which this filter is associated.
       -
         name: physical_filter
         type: string
+        length: 8
```

Comment:
Is 8 characters enough for everyone?

Reply:
Most filter names are single characters, and physical filters may contain instrument prefixes or version suffixes, that'll still be pretty small. That said, maybe the size of the instrument field + 8 would be better (i.e. 16).